examples/truffle/networks.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this supposed to get committed like this?

Reply:
Yes... the example currently deploys in rust not truffle.